lib/tasks/generate_lat_long.rake

```diff
@@ -0,0 +1,31 @@
+# lib/tasks/generate_lat_long.rake
+
+namespace :generate_lat_long do
```

Comment:
I'd namespace this "geocode" and the task "finishers"

Reply:
I'll add this now!